integration/backward_compatibility/run.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -x
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../lib/common.bash"
+
+RUNTIME=${RUNTIME:-kata-runtime}
+RUNTIME_URL=https://github.com/kata-containers/runtime
+
+# returns the latest stable version of kata-runtime
+install_latest_stable() {
+	latest="$(git ls-remote --tags "${RUNTIME_URL}"  | egrep -o "refs.*" | egrep -v "\-alpha|\-rc|{}" | cut -d/ -f3 | sort -r | head -1)"
+	"${cidir}/../../.ci/install_kata.sh" "${latest}"
+}
+
+main() {
+	runtime_path=$(get_docker_kata_path ${RUNTIME})
+	if [ -z "${runtime_path}" ] || [ "${runtime_path}" = "null" ]; then
+		die "${RUNTIME} has not been configured as a runtime in docker"
+	fi
+
+	# run a container
+	cont_name="backward_test"
+	current="$(${runtime_path} --version)"
+	docker run -dti --name "${cont_name}" --runtime="${RUNTIME}" debian bash
+
+	install_latest_stable
+
+	# stop the container
+	latest="$(${runtime_path} --version)"
+	docker rm -f "${cont_name}"
+
+	# versions must be different
+	[ "${current}" != "${latest}" ]
```

Comment:
Shouldn't this test revert to the version installed at the start of the test rather than leaving the latest stable version installed on the system?

Also, after the container has been stopped, how about:

- Creating a container with the latest stable release.
- Resetting back to the original version of Kata.
- Attempting to stop the container created with the latest stable version with the original version as an extra test?